images/inference-services/ovms/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine
+
+ENV KUBE_LATEST_VERSION="v1.20.5"
+ENV YTT_LATEST_VERSION="v0.36.0"
+
+RUN apk add --update ca-certificates \
+  && apk add --update curl \
```

Comment:
this could be just one line, which also can use `--no-cache` so you would not need to run `rm /var/cache/apk/*`. Like:

`apk add --update --no-cache ca-certificates curl`

Reply:
great tip, thank you !